common/app/conf/switches.scala

```diff
@@ -403,6 +403,11 @@ object Switches {
     safeState = On, sellByDate = never
   )
 
+  val FacebookAppLinksSwitch = Switch("Feature", "facebook-applinks",
+    "If this switch is on then shared links on the facebook mobile app will be opened in the native app instead of the mobile browser",
+    safeState = Off, sellByDate = never
```

Comment:
And there is 100% a "sell by date" - that forces us to go back and make a decision on this. It forces us to evaluate it.

A month should be long enough